Motorcycling 101 - draft doc

selfsk opened the pull request:
- motorcycling 101 - focus on what should be taught
 - added some links to about page

selfsk committed:
Trying to organize thoughts in more reasonable way

 - motorcycling 101 - focus on what should be taught
 - added some links to about page

selfsk committed:
moved motorcycling 101 into posts

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+# 101?
+
+In this post I'm trying to share my vision of "motorcycling 101", or "what I think should be taught to people who starts riding motorcycle for the first time". I think current approach to preparing brand new people for riding motorcycle is [somewhat confusing...](/motorcycles/education/learning/2021/03/18/how-i-didnt-become-an-instructor.html). I'm trying to focus on fundamentals, since lack of f ...
```
```suggestion
In this post I'm trying to share my vision of "motorcycling 101", or "what I think should be taught to people who start riding motorcycles for the first time". I think current approach to preparing brand new people for riding motorcycles is [somewhat confusing...](/motorcycles/education/learning/2021/03/18/how-i-didnt-become-an-instructor.html). I'm trying to focus on fundamentals, since the lack of focus in that area I noticed in Basic Riding Course.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+In this post I'm trying to share my vision of "motorcycling 101", or "what I think should be taught to people who starts riding motorcycle for the first time". I think current approach to preparing brand new people for riding motorcycle is [somewhat confusing...](/motorcycles/education/learning/2021/03/18/how-i-didnt-become-an-instructor.html). I'm trying to focus on fundamentals, since lack of f ...
+
+Basics and foundation of riding a motorcycle is not harder than riding a bicycle. Complexity is coming from each input bike input(things like throttle, brakes, lean angle, steering) as wells as rider's inputs (like vision, hearing, feeling), each can be adjusted within very broad spectrum and with different granularity!
```
```suggestion
The basics and foundation of riding a motorcycle is not harder than riding a bicycle. Complexity is coming from each bike's input (things like throttle, brakes, lean angle, steering) as well as rider's input (like vision, hearing, feeling), each can be adjusted within a broad spectrum and with different granularity!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+# New rider
+
+I think the most exciting thing about being a new riders is to experience things for a first time! Things like accelerating over 40mph (yes, 40mph will seem like a warp-speed), braking or leaning bike over 20 degrees. This excitement is a great feeling, remember it :) and prepare to start working on keeping this excitement in check! 
```
```suggestion
I think the most exciting thing about being a new rider is to experience things for the first time! Things like accelerating over 40mph (yes, 40mph will seem like a warp-speed), braking or leaning a bike over 20 degrees. This excitement is a great feeling, remember it :) and prepare to start working on keeping this excitement in check! 
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+I think the most exciting thing about being a new riders is to experience things for a first time! Things like accelerating over 40mph (yes, 40mph will seem like a warp-speed), braking or leaning bike over 20 degrees. This excitement is a great feeling, remember it :) and prepare to start working on keeping this excitement in check! 
+
+Over time this excitement subside and you start noticing other things, like what you feel through you hands, feet, leg, butt, etc. You need to process that data constantly while you're riding, and even some time while you're not!
```
```suggestion
Over time the excitement subsides and you start noticing other things, like what you feel through you hands, feet, legs, butt, etc. You need to process that data constantly while you're riding, and even some time while you're not!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+Over time this excitement subside and you start noticing other things, like what you feel through you hands, feet, leg, butt, etc. You need to process that data constantly while you're riding, and even some time while you're not!
+
+`Moto-self-awareness` - new rider barely understands what's going on the bike alone, not even mentioning real road world conditions. They are starting developing moto-self-awareness, it is the same thing as people who starts doing yoga or exercise becomes more self-aware about how things work, so they can use that to improve(hopefully) their level of life, in our case we want to improve our ridin ...
```
```suggestion
`Moto-self-awareness` - new rider barely understands what's going on the bike alone, not even mentioning the real world road conditions. These are starting developing moto-self-awareness, it is the same thing as people who start doing yoga or exercising, become more aware about how things work, so they can use that to improve(hopefully) their level, in our case we want to improve our riding skills.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+`Moto-self-awareness` - new rider barely understands what's going on the bike alone, not even mentioning real road world conditions. They are starting developing moto-self-awareness, it is the same thing as people who starts doing yoga or exercise becomes more self-aware about how things work, so they can use that to improve(hopefully) their level of life, in our case we want to improve our ridin ...
+
+You're just starting your path of learning you and your motorcycle! Have fun and stay upright!
```
```suggestion
You're just starting your path of learning yourself and your motorcycle! Have fun and stay upright!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+# Stay upright
+
+Goal of any rider (of any level, any type of riding, etc.) is to **STAY UPRIGHT!** (or to keep continue riding)
```
```suggestion
The goal of any rider (of any level, any type of riding, etc.) is to **STAY UPRIGHT!** (or to keep riding)
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+Goal of any rider (of any level, any type of riding, etc.) is to **STAY UPRIGHT!** (or to keep continue riding)
+
+How are we doing it? By balancing our machine while it moves through space! It is that easy... **NOT!**
```
```suggestion
How are we doing it? By balancing our machine while it moves through space! It is that simple... **NOT!**
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+How are we doing it? By balancing our machine while it moves through space! It is that easy... **NOT!**
+
+For starters we need to learn inputs that can _control_ move of us (you and bike) through space. I put those inputs in 2 groups - `speed` and `direction`:
```
```suggestion
For starters, we need to learn inputs that can _control_motion through space. I put those inputs in 2 groups - `speed` and `direction`:
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+  * handlebars
+  * body
+
+There is one more thing... most important thing - `mental fitness`. 
```
```suggestion
There is one more thing... the most important one - `mental fitness`. 
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+There is one more thing... most important thing - `mental fitness`. 
+
+Riding a motorcycle requires additional processing and analysing of what you see(most important), hear, feel or even smell. Both input groups are based on that - lack of `mental fitness` will causes problems! Things like target fixation is one of them...
```
```suggestion
Riding a motorcycle requires additional processing and analyzing of what you see(most important), hear, feel or even smell. Both input groups are based on that - lack of `mental fitness` will causes problems! Things like target fixation is one of them...
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+Riding a motorcycle requires additional processing and analysing of what you see(most important), hear, feel or even smell. Both input groups are based on that - lack of `mental fitness` will causes problems! Things like target fixation is one of them...
+
+It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is lacking(you can not process all inputs is not on the same level(i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction no ...
```
```suggestion
It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is lacking(you cannot process all the inputs on the same level(i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction are not. Abrupt(not sudden) inputs are an example of that.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is lacking(you can not process all inputs is not on the same level(i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction no ...
+
+By using combination of these groups we are balancing our machine! **In simpler terms - bike needs to move fowrard to stay upright. As long as rider controls where and how it goes, and there is a surface to ride on, bike will be upright.**
```
```suggestion
By using combination of these groups we are balancing our machine! **In simpler terms - bike needs to move fowrard to stay upright. As long as a rider controls where and how it goes, and there is a surface to ride on, bike will be upright.**
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+Complexity of riding motorcycle comes from almost infinite scale of adjustement in each of these groups, and each item in group. It is infinite because as you learn, bar moves ahead too! 
+
+Simple analogy that I think fits here - you have a ruler with only meter/yard marks - so all that you can work with are things which are about the size of 1 meter/yard - while you're progressing in your moto-self-awareness journey,  that ruler getting replaced with one with 1cm/feet marks, further you developing - it will go even to smaller sizes - 1mm/inch and so on. Going from 1 meter/yard mark ...
```
```suggestion
Simple analogy that I think fits here - you have a ruler with only meter/yard marks - so all that you can work with are things which are about the size of 1 meter/yard - while you're progressing in your moto-self-awareness journey,  that ruler getting replaced with one with 1cm/feet marks, further you developing - it will go even to smaller sizes - 1mm/inch and so on. Going from 1 meter/yard marks to 1cm/feet can be quick, but the finer granularity you want to be with your inputs - the harder it will be to achieve this. 
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+#### Note on clutch
+
+I didn't include cluth in any of groups above because, I don't treat it as main input, but auxilliary(i.e. electric bikes doesn't have ones, neither scooters). I suggest to treat cluth as a lever that disconnects engine power from rear wheel and it's your safety lever if bike scared you!
```
```suggestion
I didn't include cluth in any of the groups above because, I don't treat it as a main input, but auxilliary(i.e. electric bikes don't have ones, neither scooters). I suggest to treat cluth as a lever that disconnects engine power from a rear wheel and it's your safety lever if bike scared you!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,60 @@
+
+I didn't include cluth in any of groups above because, I don't treat it as main input, but auxilliary(i.e. electric bikes doesn't have ones, neither scooters). I suggest to treat cluth as a lever that disconnects engine power from rear wheel and it's your safety lever if bike scared you!
+
+Important part for new rider to practice is "friction zone" - but I believe by doing slow speed drills you will practice it anyway.
```
```suggestion
Important part to practice for a new rider is "friction zone" - but I believe by doing slow speed drills you will practice it anyway.
```

selfsk committed:
Apply suggestions from code review

Co-authored-by: tashakeller <nataly.keller@gmail.com>

selfsk committed:
ispell run

selfsk committed:
Added paragraph about Training approach

selfsk committed:
more thoughts on mental fitness and DMV waiver

selfsk committed:
Added an "inspired by" part

 - some context where my ideas are coming from - I think can be helpful for further understanding

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+# 101?
+
+In this post I'm trying to share my vision of "motorcycling 101", or "what I think should be taught to people who start riding motorcycles for the first time". I think current approach to preparing brand new people for riding motorcycles is [somewhat confusing...](/motorcycles/education/learning/2021/03/18/how-i-didnt-become-an-instructor.html). I'm trying to focus on fundamentals, since the lack ...
```
```suggestion
In this post I'm trying to share my vision of "motorcycling 101", or "what I think should be taught to people who start riding motorcycles for the first time". I think current approach to preparing brand new people for riding motorcycles is [somewhat confusing...](/motorcycles/education/learning/2021/03/18/how-i-didnt-become-an-instructor.html). I'm trying to focus on fundamentals, since I've noticed the lack of focus in that area in Basic Riding Course.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+There is one more thing... the most important one - `mental fitness`. 
+
+Riding a motorcycle requires additional processing and analyzing of what you see(most important), hear, feel or even smell. Both input groups are based on that - lack of `mental fitness` will causes problems! Things like target fixation is one of them...
```
```suggestion
Riding a motorcycle requires additional processing and analyzing of what you see(most important), hear, feel or even smell. Both input groups are based on that, lack of `mental fitness` will cause problems! Things like target fixation is one of them...
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+Riding a motorcycle requires additional processing and analyzing of what you see(most important), hear, feel or even smell. Both input groups are based on that - lack of `mental fitness` will causes problems! Things like target fixation is one of them...
+
+It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is lacking(you cannot process all the inputs on the same level(i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction are no ...
```
```suggestion
It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is missing (you cannot process all the inputs on the same level (i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction are not. Abrupt(not sudden) inputs are an example of that.
```

selfsk replied:
I will keep `lacking` here, `missing` is a strong word - it's more about not having enough mental fitness, instead of not having it at all.

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+It is critical to understand that all groups are intertwined - even if you have a _good_ control over speed and direction, but mental fitness is lacking(you cannot process all the inputs on the same level(i.e. eyes, ears, hands, feet) at the same rate) - it's easy to go over your head! It is true the other way around, your `mental fitness` can be at _good_ level, but speed and/or direction are no ...
+
+By using combination of these groups we are balancing our machine! **In simpler terms - bike needs to move forward to stay upright. As long as a rider controls where and how it goes, and there is a surface to ride on, bike will be upright.**
```
```suggestion
By using a combination of these groups we are balancing our machine! **In simpler terms - a bike needs to move forward to stay upright. As long as a rider controls where and how it goes, and there is a proper surface to ride on, bike will be upright.**
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+By using combination of these groups we are balancing our machine! **In simpler terms - bike needs to move forward to stay upright. As long as a rider controls where and how it goes, and there is a surface to ride on, bike will be upright.**
+
+Complexity of riding motorcycle comes from almost infinite scale of adjustment in each of these groups, and each item in group. It is infinite because as you learn, bar moves ahead too! 
```
```suggestion
Complexity of riding motorcycle comes from almost infinite scale of adjustment in each of these groups, and each item in group. It is infinite because as you learn, the bar moves ahead as well! 
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+Complexity of riding motorcycle comes from almost infinite scale of adjustment in each of these groups, and each item in group. It is infinite because as you learn, bar moves ahead too! 
+
+Simple analogy that I think fits here - you have a ruler with only meter/yard marks - so all that you can work with are things which are about the size of 1 meter/yard - while you're progressing in your moto-self-awareness journey,  that ruler getting replaced with one with 1cm/feet marks, further you developing - it will go even to smaller sizes - 1mm/inch and so on. Going from 1 meter/yard mark ...
```
```suggestion
Simple analogy that I think fits here - you have a ruler with only a yard marks - so all that you can work with are things which are about the size of a yard - while you're progressing in your moto-self-awareness journey,  that ruler getting replaced with one with 1 feet marks, further you developing - it will go even to smaller sizes - 1 inch and so on. Going from 1 yard marks to 1 feet can be quick, but the finer granularity you want to be with your inputs - the harder it will be to achieve this. 
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+#### Note on clutch
+
+I didn't include clutch in any of the groups above because, I don't treat it as a main input, but auxiliary(i.e. electric bikes don't have ones, neither scooters). I suggest to treat clutch as a lever that disconnects engine power from a rear wheel and it's your safety lever if bike scared you!
```
```suggestion
I didn't include clutch in any of the groups above, because I don't treat it as a main input, but auxiliary(i.e. electric bikes don't have ones, neither scooters). I suggest to treat clutch as a lever that disconnects engine power from a rear wheel and also it's your safety lever when bike scares you!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+# Training
+
+Groups of inputs - `speed` and `direction` must be trained separately, at the beginning. Then, it should be trained together - student need to understand how `speed` and `direction` work together, how one affect another. Things like trail braking, although is treated as advanced techniques, it should be mandatory. 
```
```suggestion
Groups of inputs - `speed` and `direction` must be trained separately, at the beginning. Then, it should be trained together - student must understand how `speed` and `direction` work together, how one affects another. Things like trail braking, although is treated as advanced techniques, learning this should be mandatory, in my opinion.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+Groups of inputs - `speed` and `direction` must be trained separately, at the beginning. Then, it should be trained together - student need to understand how `speed` and `direction` work together, how one affect another. Things like trail braking, although is treated as advanced techniques, it should be mandatory. 
+
+`Mental fitness` should be trained along with `speed` and `direction`. Without understanding how motorcycle moves and how to control it, students don't have the tools to train their brains. Now, student start to grasp understanding of how things work - it should help with preparing brain for processing additional data (`moto-self-awareness`).
```
```suggestion
`Mental fitness` should be trained along with `speed` and `direction`. Without understanding how motorcycle moves and how to control it, students don't have the tools to train their brains. Now, student starts to grasp understanding of how things work - it should help with preparing brain for processing additional data (`moto-self-awareness`).
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+`Mental fitness` should be trained along with `speed` and `direction`. Without understanding how motorcycle moves and how to control it, students don't have the tools to train their brains. Now, student start to grasp understanding of how things work - it should help with preparing brain for processing additional data (`moto-self-awareness`).
+
+Training should be going a bit longer, too. I think each day should focus on different groups, and only after each group is somewhat familiar to student, we can start combining both, and 3th and 4th days should be practice of fundamentals with different drills and final evaluation. Yes, "evaluation" not a test - after course student should get a feedback, which can be used for further learning.
```
```suggestion
Training should be going a bit longer, too. I think each day you should focus on different groups, and only after each group is somewhat familiar to a student, we can start combining both, and 3rd and 4th days should be practice of fundamentals with different drills and final evaluation. Yes, "evaluation", not a test - after course student should get a feedback, which can be used for further learning.
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+Training should be going a bit longer, too. I think each day should focus on different groups, and only after each group is somewhat familiar to student, we can start combining both, and 3th and 4th days should be practice of fundamentals with different drills and final evaluation. Yes, "evaluation" not a test - after course student should get a feedback, which can be used for further learning.
+
+After this course, student should be able to pass a DMV exam with no problem at all. DMV waiver shouldn't be a bonus, insurance discounts - absolutely!
```
```suggestion
After this course, a student should be able to pass a DMV exam with no problem at all. DMV waiver shouldn't be a bonus, insurance discounts - absolutely!
```

tashakeller commented on _posts/2021-04-13-motorcycling-101.md:
```diff
@@ -0,0 +1,73 @@
+
+After this course, student should be able to pass a DMV exam with no problem at all. DMV waiver shouldn't be a bonus, insurance discounts - absolutely!
+
+Bottom line BRC must be stopped being a factory of DMV waivers, and become a factory that should inspire new riders to pursue further learning and practicing. It should be showing what the possibilities are, and where starting rider is and put them on path of self education. Otherwise it must be a very thorough process which couldn't be done in 12 hours.
```
```suggestion
Bottom line BRC must be stopped being a factory of DMV waivers, and become an experience that inspires new riders to pursue learning and practicing further. It should be showing what the possibilities are, where newbie rider is, and put him/her on a path of self education. Otherwise, it must be a very thorough process which couldn't be completed in 12 hours.
```

selfsk committed:
Update _posts/2021-04-13-motorcycling-101.md

Co-authored-by: tashakeller <nataly.keller@gmail.com>

selfsk committed:
Update _posts/2021-04-13-motorcycling-101.md

Co-authored-by: tashakeller <nataly.keller@gmail.com>

selfsk committed:
Apply suggestions from code review

Co-authored-by: tashakeller <nataly.keller@gmail.com>
